src/components/_global/BalTable/BalTable.vue

```diff
@@ -14,44 +14,50 @@
             v-for="(column, columnIndex) in filteredColumns"
             :key="`header-${column.id}`"
             :class="[
-              'p-6 bg-white headingShadow border-b border-gray-200',
+              'p-6 bg-white headingShadow border-b border-gray-100',
               column.className,
-              column.align === 'right' ? 'text-right' : 'text-left',
               getHorizontalStickyClass(columnIndex),
               isColumnStuck ? 'isSticky' : '',
               column.sortKey ? 'cursor-pointer' : ''
             ]"
             :ref="setHeaderRef(columnIndex)"
             @click="handleSort(column.id)"
           >
-            <slot
-              v-if="column.Header"
-              v-bind="column"
-              :name="column.Header"
-            ></slot>
-            <div v-else>
-              <h5 class="text-base font-semibold text-gray-800">
-                {{ column.name }}
-              </h5>
+            <div
+              :class="[
+                'flex',
+                column.align === 'right' ? 'justify-end' : 'justify-start'
+              ]"
+            >
+              <slot
+                v-if="column.Header"
+                v-bind="column"
+                :name="column.Header"
+              ></slot>
+              <div v-else>
+                <h5 class="text-base font-semibold text-gray-800">
+                  {{ column.name }}
+                </h5>
+              </div>
+              <BalIcon
+                name="arrow-up"
+                size="sm"
+                v-if="
+                  currentSortColumn === column.id &&
+                    currentSortDirection === 'asc'
+                "
+                class="ml-1 flex items-center"
+              />
```

Comment:
this whole block has already been merged to develop so I am not sure why its showing up here